packages/test/test-end-to-end-tests/src/test/gc/gcDatastoreAliased.spec.ts

```diff
@@ -83,7 +83,9 @@ describeFullCompat("GC Data Store Aliased Full Compat", (getTestObjectProvider)
 		const containerRuntime2 = mainDataStore2._context
 			.containerRuntime as unknown as IContainerRuntime;
 		assert.doesNotThrow(
-			async () => containerRuntime2.getRootDataStore(alias),
+			async () =>
+				containerRuntime2.getAliasedDataStore?.(alias) ??
```

Comment:
We need the fallback here because of ContainerRuntime / DataStoreRuntime compat config?

Reply:
If so, maybe put a comment saying in what version the fallback can be removed.

Reply:
No, this is because when the test runs in N-1 compat mode, the container runtime will be an older version and will not have `getAliasedDataStore` API

Reply:
Not sure if a comment is needed. The fallback can be removed once the getRootDataStore API is removed. That has to follow the back-compat rule.